makefile

```diff
@@ -132,6 +132,7 @@ test: bin/M2-Planet | bin test/results
 .PHONY: Generate-test-answers
 Generate-test-answers:
 	sha256sum test/results/* >| test/test.answers
+	sha256sum test/test1000/proof >| test/test1000/proof.answer
```

Comment:
I guess it's fine but note that you'll still need to run it twice.

Generally I expect test 1000 to fail on changes but if you run Generate-test-answers then, you wouldn't have correct checksums for anything that comes after test1000 (which is usually other arches).

Reply:
That's also how I do it now. I basically run `make clean test; make Generate-test-answers;  sha256sum test/test1000/proof > test/test1000/proof.answer` twice and then check for the `OK`s.